<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QimsysSettingsWidget</class>
 <widget class="QWidget" name="QimsysSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>308</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="icon">
     <property name="text">
      <string>Icon</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="name">
     <property name="text">
      <string>Qimsys Plugin</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="authorLabel">
     <property name="text">
      <string>Author:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="author">
     <property name="text">
      <string>Author</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="urlLabel">
     <property name="text">
      <string>Url:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="url">
     <property name="text">
      <string>http://</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="descriptionLabel">
     <property name="text">
      <string>Description:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QTextBrowser" name="description"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="translatorLabel">
     <property name="text">
      <string>Translator:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="translator">
     <property name="text">
      <string>Translator</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="licenseLabel">
     <property name="text">
      <string>License:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QTextBrowser" name="license"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
